arch/arm/dts/rzpi.dts

```diff
@@ -13,6 +13,20 @@
 #include <dt-bindings/pinctrl/rzg2l-pinctrl.h>
 #include <configs/rzpi.h>
 
+// Total DDR size 
+#if (ISSI_4Gb_DDR)
+#define DDR_TOTAL_SIZE    0x20000000  
+#elif (ISSI_8Gb_DDR)
+#define DDR_TOTAL_SIZE    0x40000000  
+#else
+#  error Missing memory side/type define
+#endif
+
+#define DDR_BASE_ADDR    0x48000000
+// 128MB is reserved for secure area and 128MB is reserved for ECC memory 
+#define DDR_RESERVED_SIZE 0x10000000  // 256MB reserved (128MB secure + 128MB ECC)
+#define DDR_AVAILABLE_SIZE (DDR_TOTAL_SIZE - DDR_RESERVED_SIZE)
+
```

Comment:
@nam-vo-khac-rvc ,

Can you move the definition to "include/configs/rzpi.h"? Also, check for duplicated values.

Reply:
Updated in the commit: 147e0db77c1d0571df468de020ce51836677b4bc